paddle/operators/add_two_op.cu

```diff
@@ -14,7 +14,7 @@
 
 #define EIGEN_USE_GPU
 #include "paddle/framework/op_registry.h"
-#include "paddle/operators/add_op.h"
+#include "paddle/operators/add_two_op.h"
```

Comment:
Curious why rename to `add_two_op`?

Reply:
Because `REGISTER_OP(add_two, ops::AddOp, ops::AddOpMaker, add_two_grad, ops::AddOpGrad);` in `add_op.cc`, and `USE_OP(add_two)` in `pybind.cc`

Reply:
I see.

Reply:
`add_two_op` seems odd, `add_op` may be more clear? Since we have `mul_op`.

Reply:
I agree, then I will revert and change `add_two` in `REGISTER_OP` and `USE_OP` to be `add`.

Reply:
Thanks very much!